src/commands/profile/profileAbout.ts

```diff
@@ -0,0 +1,63 @@
+import { Message, MessageEmbed, User } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import _ from 'lodash';
+import { UserProfile, getUserProfileById } from '../../components/profile';
+import { getCoinBalanceByUserId } from '../../components/coin';
+import { BaseCommand } from '../../utils/commands';
+
+enum prettyProfileDetails {
+  about_me = 'About Me',
+  birth_date = 'Birth Date',
+  preferred_name = 'Preferred Name',
+  preferred_pronouns = 'Preferred Pronouns',
+  term = 'Term',
+  year = 'Year',
+  faculty = 'Faculty',
+  program = 'Program',
+  last_updated = 'Last Updated'
+}
+
+class UserProfileAboutCommand extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'about',
```

Comment:
For this (and all future) functionality, try to name commands in the form of .[group][subcommand], like how the interviewer commands are set up.

It helps make sure no future commands overlap on triggers.

Additionally, consider adding a default command to the .profile command, so that the command group has a default behaviour. Something such as how .interviewers is identical to .interviewerslist

Reply:
A little confused over the second part of this, .interviewers is an alias for .interviewslist. Do you want me to rename profileAbout to profile and then have the name be profile with alias profile-about like so?

<img width="762" alt="image" src="https://user-images.githubusercontent.com/36215359/155937682-6323f7cb-225a-4a3c-a8da-03b65158e0e7.png">

Reply:
so basically, 

![image](https://user-images.githubusercontent.com/37851975/156075173-591edde1-f546-47df-87c8-2e1ec32b909d.png)

every single class begins with [group]
and there is an interviewers.ts which is exactly the list function (there is no interviewerList.ts

in ur case, since profiles is ur group name, every command should start with ``profiles`` but probably ``profile`` instead.
there should be a profiles.ts that is exactly profilesCommand (what u have) and aliases are about-profile, profile abt, etc